src/components/NoClubsFound.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { Button, Card, Heading, Image, Link } from "@hackclub/design-system";
+
+const Base = Card.extend.attrs({
+    bg: "snow",
+    boxShadowSize: "lg",
+    p: 4
+})`
+    align-items: center;
+    flex-direction: column;
+    max-width: 48rem;
+`;
+
+const SuperButton = Button.withComponent(Link).extend`
```

Comment:
remove the `.withComponent(Link)`, that's for links in the same Gatsby site

Reply:
I was using the one from DS: https://github.com/hackclub/design-system/blob/master/src/Link.js

Reply:
oh! sorry, yeah. buttons are a tags by default, Button.button will give you a real button tag